packages/theme-data/src/baseTheme/components/textLink.js

```diff
@@ -0,0 +1,22 @@
+import { COLOR, LENGTH } from "../../consts/types";
+
+export default {
+  "textLink.textColor": {
+    type: COLOR,
+    value: "#006EAF"
```

Comment:
I think we might want to add these to `basics.colors` (light and dark themes below). This could be an example: https://github.com/Autodesk/hig/blob/development/packages/theme-data/src/basics/colors.js#L63

From reading the HIG documentation, maybe we could do `textLink.lightTheme.color` and `textLink.darkTheme.color`.